GVFS/GVFS.Upgrader/UpgradeOrchestrator.cs

```diff
@@ -14,6 +15,8 @@ public class UpgradeOrchestrator
         private const EventLevel DefaultEventLevel = EventLevel.Informational;
 
         private ProductUpgrader upgrader;
+        private string logDirectory = ProductUpgraderInfo.GetLogDirectoryPath();

```

Comment:
Should we keep `logDirectory` around?  If it's always going to be `ProductUpgraderInfo.GetLogDirectoryPath()` we could go back to using that directly.

Reply:
Yeah - that is an option. My thinking was that instead of sprinkling the calls to `ProductUpgraderInfo.GetLogDirectoryPath` around in multiple places, I could consolidate this to a single instance variable, and reduce calls to static methods. It is not strictly needed for this change (but now we do use the `logDirectory` in multiple places.